layouts/case-studies/single.html

```diff
@@ -2,6 +2,37 @@
 {{ with .Params.content_url }}
 <meta http-equiv="refresh" content="0; URL='{{ . }}'" />
 {{ else }}
-	{{ .Content }}
+	{{ if .Params.new_case_study_styles }}
```

Comment:
This param is used for backward compatibility with the current styles. Can be removed later (in other PR) once all case study pages has been update.